integr8ly/integreatly-operator
# AMO 1.8.0 support

# Description
This pr updates AMO to v1.8.0 

## Verification
- provision cluster
- Install rhoam using local installation steps in README using this [branch](https://github.com/jackdelahunt/integreatly-operator/tree/AMO-1-7-0).
- Verify prometheus operator image is `quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:19ecf8f3cb62545bc81eeddd3ef8adc105ec5bad5538ee556aade7eb8f9d6dbf` by going to Projects > Middleware Monitoring Operator > Pods > prometheus-operator-... > Container details 


## Type of change




## Checklist
